src/main/kotlin/no/nav/klage/oppgave/clients/OppgaveClient.kt

```diff
@@ -33,6 +34,7 @@ class OppgaveClient(
 
     }
 
+    @Retryable
```

Comment:
Burde denne vært på `getOnePage(...)`? Funker vel ikke på private-metoder, men kan kanskje åpnes opp litt?

Reply:
Jeg er i og for seg enig i det, men da må vi endre koden en god del. Det er ikke bare at det ikke fungerer på private-metoder men det fungerer ikke på kall internt i en klasse heller. Så vi må splitte det opp i to, dra noe av det ut i OppgaveService kanskje?

Reply:
Har innført OppgaveRepository